src/base/_defaults.scss

```diff
@@ -51,14 +51,19 @@ ul {
  * 1. Forces the viewport to be filled. Necessary for the footer to
  *    "stick" to the bottom of short pages.
  *    @see https://css-tricks.com/couple-takes-sticky-footer/#there-is-grid
- * 2. Keep full-bleed items from triggering a horizontal scrollbar.
+ */
+html {
+  block-size: 100%; // 1
+}
+
+/**
+ * 1. Keep full-bleed items from triggering a horizontal scrollbar.
  *    @see https://cloudfour.com/thinks/breaking-out-with-viewport-units-and-calc/
  */
 
 html,
 body {
-  block-size: 100%; // 1
```

Comment:
This change was needed because `body` needed to not have `block-size: 100%` because it already had `min-block-size: 100%` in the page object.

I don't know why the combination of both `min-block-size` and `block-size` (set to the same value) made it have a second scrollbar.